src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -109,6 +109,16 @@ extension DynamicTableViewController {
 			view?.titleLabel.accessibilityLabel = title
 			view?.titleLabel.accessibilityTraits = .header
 			view?.title = title
+			
+			// add optional background gradient
+			if let backgroundGradient = backgroundGradient, let view = view {
```

Comment:
I think it would be nicer to have this inside of `DynamicTableViewHeaderImageView`. There could be a property `backgroundView` to avoid searching for subviews.

Reply:
The problem with putting it in `DynamicTableViewHeaderImageView` is that it has a title label that is placed above the the image which we don't need in this case.

Reply:
I would not expect that the label is overlapped by the backgroundGradient if I use it.

Reply:
I mean that the title label occupies space above the image which conflicts with the layout need in this case.